docs/index.md

```diff
@@ -171,6 +171,13 @@ user-guides/faq
 
 ```
 
+```{toctree}
+:caption: How-to
+:hidden:
+
+how-to/jcloud_deploy
```

Comment:
do not add new section. Do not change the on big structure of the docs, put into server section